### PR TITLE
Added animations

### PR DESCRIPTION
**Pull request by wbrbr**

### CODE REVIEW

File: src/animation.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    anim->frameHeight = frameHeight;
+    anim->current_frame = 0;
+    anim->elapsedTime = 0;
+    anim->timeBetweenFrames = timeBetweenFrames;
+    anim->startX = startX;
+    anim->startY = startY;
+    return anim;
+}
+
+void updateAnimation(animation* anim,float deltaTimeS,int texX,int texY)
```
Comment (vhakulinen) [inline src/animation.c]: Unused parameters `texX` and `texY`

File: src/animation.c [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#include "animation.h"
+#include <SDL2/SDL_image.h>
+
+animation* CreateAnimation(SDL_Renderer *ren,char *imagePath,int frameNumber,int frameWidth,int frameHeight,float timeBetweenFrames,int startX,int startY)
```
Comment (vhakulinen) [inline src/animation.c]: Unused parameters `*ren` and `imagePath`

**Comment by vhakulinen** [comment]
And please, use spaces after `,`.

**Comment by wbrbr** [comment]
Thanks for the review, I removed the unnecessary parameters.

**Pull request merged**